### DISCUSSION TITLE
JWT Token user has picture instead of image

### DISCUSSION THREAD

**Discussion by laurenzhonauer** [nextauthjs/next-auth#3231] [Help]
### Description 🐜

When enabling jwt in the nextauth config like so

```
session: {
    jwt: true,
    maxAge: 30 * 24 * 60 * 60, //30 days,
  }
```

The `user` object in the session object is not containing the profile picture as `image` but as `picture`

### Is this a bug in your own project?

Yes

### How to reproduce ☕️

In pages/api/auth/[...nextauth] configure

```
session: {
    jwt: true,
    maxAge: 30 * 24 * 60 * 60, //30 days,
}
```

### Screenshots / Logs 📽

Session when enabling jwt:

```{
expires: "2021-12-18T14:22:30.559Z"
user: {
  email: "use.mail@mail.com"
  exp: 1639837348
  iat: 1637245348
  name: "laurenzhonauer"
  picture: "https://avatars.githubusercontent.com/u/44588135?v=4"
  sub: "8b754a31-72d6-4367-a30d-01c6beff592d
  }
}
```

Session when not enabling jwt

```
{
accessToken: "3981375329d00f71b614f25b883cf35112ca1fef7ece15b6c1f85ac00b9a1819"
expires: "2021-12-18T14:24:25.058Z"
user: {
  createdAt: "2021-11-18T14:22:28.672Z"
  email: "use.mail@mail.com"
  emailVerified: null
  id: "8b754a31-72d6-4367-a30d-01c6beff592d"
  image: "https://avatars.githubusercontent.com/u/44588135?v=4"
  name: "laurenzhonauer"
  tokens: 0
  updatedAt: "2021-11-18T14:22:28.673Z"
  }
}
```

### Environment 🖥

System:
    OS: Linux 5.10 Ubuntu 20.04.3 LTS (Focal Fossa)
    CPU: (5) x64 Intel(R) Core(TM) i5-8350U CPU @ 1.70GHz
    Memory: 1.41 GB / 3.84 GB
    Container: Yes
    Shell: 5.0.17 - /bin/bash
  Binaries:
    Node: 16.6.0 - ~/.nvm/versions/node/v16.6.0/bin/node
    npm: 7.20.5 - ~/.nvm/versions/node/v16.6.0/bin/npm
  npmPackages:
    next: 12.0.2 => 12.0.2 
    next-auth: ^3.29.0 => 3.29.0 
    react: 17.0.2 => 17.0.2

### Contributing 🙌🏽

No, I am afraid I cannot help regarding this

**Comment by marytal** [comment]
Hi! :) Any updates on this? My `user` from the `useSession` hook shows a `null` image even though the DB value is set. I am using a JWT session and I see that it has a "picture" value instead. Wondering if this is the cause.

Cheers!
